documentation/src/main/asciidoc/topics/ref_cache_store_sfs.adoc

```diff
@@ -19,9 +19,26 @@ defragmented.
 
 .Declarative configuration
 
+* For embedded deployments, {brandname} stores data to the current working
+directory. To store data in a different location, specify the absolute path to
+that directory as the value, as in the following example:
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<persistence>
+   <file-store path="/path/to/myDataStore" max-entries="5000"/>
```

Comment:
This isn't correct on master anymore due to https://issues.redhat.com/browse/ISPN-11556. If the value of `path` is an absolute path, then that value must be a sub path of `globalState.persistentLocation` or an exception is thrown.

For example:
```java
// Global configuration
globalState().persistentLocation(`/tmp/example/`);
...
// Cache configuration
1. config.addSingleFileStore().location('/tmp/example/file-store-absolute'`)
2. config.addSingleFileStore().location('file-store-relative'`)
3. config.addSingleFileStore().location('/example/`)
```

Example 1 and 2 are valid and result in a file-store being created at `/tmp/example/file-store-absolute` and `/tmp/example/file-store-relative`.

Example 3 is invalid as the root is different, i.e. `/example` != `/tmp/example` and so a [`CacheConfigurationException`](https://github.com/infinispan/infinispan/blob/ecb0ff73fbc8df9898c96a79c4c300d44c239c0c/core/src/main/java/org/infinispan/util/logging/Log.java#L1891-L1892) is thrown.

Reply:
So if the `path` and `relative-to` attributes in a configuration xml result in a path that is absolute and not a child of `globalstate.persistenceLocation`, an exception will be thrown.

Reply:
thanks @ryanemerson ! shall I adjust for 10.1.x and push the docs commit to the backport for ISPN-11556? I think I just need to remove the ConfigSingleFileStoreRemote.java example from 10.1.x.

Reply:
@oraNod why omit `ConfigSingleFileStoreRemote`? That is still valid in 10.1.x

Reply:
you're right, thanks. I didn't see the backport when I commented. that makes it easier.